features/admin.groups.v1/components/edit-group/edit-group.tsx

```diff
@@ -139,8 +139,10 @@ export const EditGroup: FunctionComponent<EditGroupProps> = (props: EditGroupPro
 
         getUsersList(null, null, null, null, userstore)
             .then((response: UserListInterface) => {
-                setUsersList(response.Resources);
-                setSelectedUsersList(filterUsersList([ ...response.Resources ]));
+                if (response.Resources?.length > 0) {
+                    setUsersList(response.Resources);
+                    setSelectedUsersList(filterUsersList([ ...response.Resources ]));
+                }
```

Comment:
IMO this logic should be moved to the `filterUsersList` function

Reply:
because from getUsersList it can return a null or empty value and checking of that should be done when setting the users list instead of setting them if its null.

That logic should be move inside the `filterUsersList` function

Reply:
if there are no users, Resources array is not returned so it will be undefined. In this case spread operation in line 144 will throw an error. We can move the array length validation inside the `filterUserList` function and keep the undefined check for Resources object in the same place. WDYT?